src/main/java/com/openlattice/util/Singletons.java

```diff
@@ -0,0 +1,29 @@
+package com.openlattice.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public final class Singletons {
```

Comment:
java 😢

Reply:
Yeah, it's just not very elegant for Java classes to use references to Kotlin primitive arrays and Kotlin has no need for the Object arrays with `emptyArrayOf()` so this is primarily for Java classes

Reply:
(and kotlin classes that use empty primitive arrays)